app/javascript/clock.js

```diff
@@ -0,0 +1,24 @@
+function mueveReloj(){
+  momentoActual = new Date()
+  hora = momentoActual.getHours()
+  minuto = momentoActual.getMinutes()
+  segundo = momentoActual.getSeconds()
+
+  str_segundo = new String (segundo)
+  if (str_segundo.length == 1)
+     segundo = "0" + segundo
+
+  str_minuto = new String (minuto)
+  if (str_minuto.length == 1)
+     minuto = "0" + minuto
+
+  str_hora = new String (hora)
+  if (str_hora.length == 1)
+     hora = "0" + hora
```

Comment:
As a suggestion you can use [Js template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)